pkg/services/object/transport_splitter.go

```diff
@@ -0,0 +1,210 @@
+package object
+
+import (
+	"bytes"
+	"context"
+
+	"github.com/nspcc-dev/neofs-api-go/v2/object"
+	"github.com/nspcc-dev/neofs-api-go/v2/refs"
+	"github.com/pkg/errors"
+)
+
+const (
+	GRPCPayloadChunkSize = 1024 * 1024 * 3 // 4 MiB is a max limit, 3 MiB should be okay
```

Comment:
Why not 4 then?

Reply:
> Why not 4 then?

Because there are other fields in the response message aside of payload in the body.